zap/src/main/java/org/zaproxy/zap/extension/search/SearchPanel.java

```diff
@@ -658,13 +658,19 @@ private JComboBox<SearchOption> getSearchType() {
                     new SearchOption(
                             Constant.messages.getString("search.toolbar.label.type.tag"),
                             ExtensionSearch.Type.Tag));
+            searchType.addItem(
+                    new SearchOption(
+                            Constant.messages.getString("search.toolbar.label.type.note"),
+                            ExtensionSearch.Type.Note));
         }
         searchType.addActionListener(
                 e -> {
                     ExtensionSearch.Type selectedopt =
                             ((SearchOption) searchType.getSelectedItem()).getType();
-                    // Inverse matching not enabled for Tags
-                    getChkInverse().setEnabled(!selectedopt.equals(ExtensionSearch.Type.Tag));
+                    // Inverse matching not enabled for Tags and Notes
```

Comment:
Search for messages that do not have a specific note seems useful.

Reply:
Agreed. I meant to implement it in a second time. I wanted to get basic note searching to work first as I need to study a bit more your codebase for inverse searching.

Inverse note searching will be part of this PR.

Reply:
Fixed in 1efbf2860.

Reply:
Done in 788de1353.